index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+        </li>
+        <li>Set |initialGlobalState|[`"unitFactorMassExponent"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"unitFactorTimeExponent"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"unitFactorTemperatureExponent"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"unitFactorCurrentExponent"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"unitFactorLuminousIntensityExponent"`]
+        to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"logicalMinimum"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"logicalMaximum"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"physicalMinimum"`] to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"physicalMaximum"`] to 0.
+        </li>
+        <li>Push |initialGlobalState| onto |globalStack|.
+        </li>
+        <li>Repeat for each |item| of |items|:
+          <div class="note">
+            <p>
+              [=Main items=] are described in the <a href=
+              "https://www.usb.org/document-library/device-class-definition-hid-111">
+              Device Class Definition for HID 1.11</a> sections 6.2.2.4 through
+              6.2.2.6. An item with the <dfn>Input tag</dfn>, <dfn>Output
+              tag</dfn>, or <dfn>Feature tag</dfn> defines a field within an
+              input, output, or feature report. The [=item data field=] of a
+              [=Main item=] with an [=Input tag=], [=Output tag=], or [=Feature
+              tag=] is a bit field containing several boolean parameters
+              describing the data field. An item with the <dfn>Collection
+              tag</dfn> defines a new collection nested within the current
+              collection, or a new [=top-level collection=] if there is no
+              current collection. The [=item data field=] of a [=Main item=]
+              with a [=Collection tag=] contains the <dfn>collection
+              type</dfn>. An item with the <dfn>End Collection tag</dfn> closes
+              the current collection.
+            </p>
+          </div>
+          <ol>
+            <li>If |item| is a [=Main item=]:
+              <ol>
+                <li>If |item| has the [=Input tag=], [=Output tag=], or
+                [=Feature tag=]:
+                  <ol>
+                    <li>Let |globalState| be the last item of |globalStack|.
+                    </li>
+                    <li>Let |reportId:octet| be |globalState|[`"reportId"`].
+                    </li>
+                    <li>Let |reportItem:HIDReportItem| be the result of
+                    [=creating a HID report item=] with |item|, |localState|,
+                    |globalState|, |usages|, and |strings|.
+                    </li>
+                    <li>For each |collection:HIDCollectionInfo| of |ancestors|:
+                      <ol>
+                        <li>Let |reports:sequence&lt;HIDCollectionInfo&gt;|
+                        be:<br>
```

Comment:
Use an &lt;ul> tag to list these cases instead of line breaks.

Reply:
Done